### DISCUSSION TITLE
Connecting MRTK3 with NORMCORE

### DISCUSSION THREAD

**Discussion by MatiasBrunacci** [MixedRealityToolkit/MixedRealityToolkit-Unity#440] [Ideas]
Is there a plan or way to connect MRTK3 with Normcore to send and receive hand-tracking features and interactions over a network?

**Comment by AMollis** [comment]
There are currently no plans to support networking solutions with MRTK.  

However, you can look at this [Azure Remote Rendering sample](https://github.com/Azure/azure-remote-rendering/tree/master/Unity/Showcase).  It serializes head poses, joint poses, and other data and sends it over Photon.  You can probably swap out Photon for Normcore.

The ARR Showcase collaboration framework is at https://github.com/Azure/azure-remote-rendering/tree/master/Unity/Showcase/App/Assets/App/Sharing

(Unfortunately, the collaboration framework uses MRTK2's extension model.  So that would also need to be updated)
